matrix/all-in-one.yaml

```diff
@@ -0,0 +1,534 @@
+component: Splunk Distribution of OpenTelemetry .NET
+version: 1.0.2
+dependencies:
+  - link: https://github.com/open-telemetry/opentelemetry-dotnet
```

Comment:
link? href? Not sure now. Would need to check how others are naming such things.

Reply:
Either is fine by me.

Reply:
022e5a729d2f179883ae837ee956a41f0e7e1a99 12ce8cc8e2454821554cafc936865f8388e5fe5d